File: ED-FPiñeiroA-A06/Node.cpp
#include<iostream>
#include "Node.h"

using namespace std;

Node::Node(){

}

Node::Node(int dato, Node* siguientet) {
	dato = dato;
	siguiente = siguiente;
}

int Node::getDato() {
	return dato;
}

void Node::setDato(int dato) {
	dato = dato;
}


Node* Node::getSiguiente() {
	return siguiente;
}

void Node::setSiguiente(Node* siguiente) {
	siguiente = siguiente;
}

File: Node.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author franc
 */
public class Node {
    int dato;
    Node siguiente;
    
    public Node(){
       
    }
    public Node(int dato, Node siguiente){
        this.dato = dato;
        this.siguiente = siguiente;
    }

    public int getDato() {
        return dato;
    }
    
    public void setDato(int dato) {
        this.dato = dato;
    }

    public Node getSiguiente() {
        return siguiente;
    }

    public void setSiguiente(Node siguiente) {
        this.siguiente = siguiente;
    }
}